components/log-viewer-webui/client/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "log-viewer-webui-client",
   "version": "0.1.0",
   "description": "",
-  "main": "src/index.jsx",
+  "main": "src/index.tsx",
   "scripts": {
     "build": "webpack --define-process-env-node-env production",
```

Comment:
The `--define-process-env-node-env` is renamed as `--config-node-env` with no documentation (not even in the changelog; their manual still reports the option. )

Reported to the webpack-cli repo: https://github.com/webpack/webpack-cli/issues/4360